src/Message/AbstractRequest.php

```diff
@@ -1075,6 +1078,24 @@ protected function buildPaymentMethod($paymentMethodType, $addAttributes = false
             if (!$this->isUpdate()) {
                 $paymentMethod->type = self::PAYMENT_METHOD_PAYPAL;
             }
+        } elseif ($paymentMethodType === self::PAYMENT_METHOD_APPLE_PAY && $card !== null) {
+            $applePay = new stdClass();
+
+            /**
+             * @var \Omnipay\Vindicia\NonStrippingCreditCard $card
+             */
+            $applePay->paymentInstrumentName = $card->getPaymentInstrumentName();
+            $applePay->paymentNetwork = $card->getPaymentNetwork();
+            $applePay->paymentData = $card->getToken();
+            $applePay->transactionIdentifier = $card->getTransactionIdentifier();
+            $applePay->expirationDate = $card->getExpiryDate('Ym');
```

Comment:
`'Ym'` is used above for credit cards also. could you make that a const?

Reply:
Making it a const outside of those if statements causes a lot of the tests to fail. For some tests, `$card->getExpiryDate('Ym')` is returning null because the card doesn't exist.

Reply:
i just mean making a const literally for `'Ym'`. Called something like VINDICIA_EXPIRATION_DATE_FORMAT